app/services/participants/change_schedule.rb

```diff
@@ -8,7 +8,7 @@ class ChangeSchedule < Action
 
     validates :schedule_identifier, presence: true
     validate :validate_new_schedule_found
-    # validates :cohort, npq_contract_for_cohort_and_course: true # TODO we don't have NPQ Contract yet
+    validates :cohort, contract_for_cohort_and_course: true
```

Comment:
having a think about this, do we really need it here, in ECF it's for funding cap, here we get the cap from cohort so maybe we don't need to check the contract after all?